bleak/backends/winrt/client.py

```diff
@@ -113,10 +113,14 @@ class BleakClientWinRT(BaseBleakClient):
     a package that enables Python developers to access Windows Runtime APIs directly from Python.
 
     Args:
-        address_or_ble_device (`BLEDevice` or str): The Bluetooth address of the BLE peripheral to connect to or the `BLEDevice` object representing it.
+        address_or_ble_device (``BLEDevice`` or str): The Bluetooth address of the BLE peripheral
+            to connect to or the ``BLEDevice`` object representing it.
 
     Keyword Args:
         timeout (float): Timeout for required ``BleakScanner.find_device_by_address`` call. Defaults to 10.0.
+        use_cached_services (bool): ``True`` allows Windows to fetch the services, characteristics and descriptors
```

Comment:
I think this would be a good time to start implementing #623 and do something to indicate that this kwarg is windows-only.

Reply:
Did I write that issue?! Ok. Let's start changing the standard now then. I will change for this method now and do the rest in the documentation rewriting.

Reply:
I tried both way presented in #623, but I didn't like either one of them now. Went with solution number two: the `win` dict in the end. It did however produce some problems documenting this in docstrings. I ended up with a list and emulating the emphasis on the dict keys to fit the RTD theme. If you have a better suggestion on how to document it, I will gladly change to that.

It looks ok in html. In the actual docstring, a bit cluttered.